PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +1,19 @@
-This pull request `[DESCRIBE WHAT THIS PULL REQUEST IS TRYING TO ACCOMPLISH HERE]`. It `[DESCRIBE HOW IT ACCOMPLISHES THE GOAL HERE]`.
+### Brief summary of changes
 
-See also: `[PROVIDE REFERENCES TO OTHER TICKETS, PULL REQUESTS, OR MAILING LIST THREADS HERE. OPTIONAL.]`
+
+
+### This resolves issue...
+
+- [] Redmine? #
+
+- [] Github? #
+
+### This PR improves LORIS by...
```

Comment:
I personally don't feel that this 'improves LORIS by' section is needed because it could easily overlap content going into 'brief summary of changes' and 'this resolves issue' sections. it would be a bit redundant